# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: serkan@gentoo.org-20081126194218-u376kxy72v0dx7dc
# target_branch: ../gtkhotkey/
# testament_sha1: 06ccd01bc7499075c62274cb25269a005b0c5efd
# timestamp: 2008-11-26 23:41:48 +0200
# base_revision_id: mikkel.kamstrup@gmail.com-20081028221117-\
#   90lhle1k6yd06j6f
# 
# Begin patch
=== modified file 'test/Makefile.am'
--- test/Makefile.am	2008-10-27 21:09:19 +0000
+++ test/Makefile.am	2008-11-26 19:42:18 +0000
@@ -11,11 +11,11 @@
 	 -Wall\
 	 -g
 
-gtk_hotkey_libs = $(top_builddir)/src/libgtkhotkey.la
+gtk_hotkey_libs = $(top_builddir)/src/libgtkhotkey.la $(GTK_HOTKEY_LIBS)
 
 helper_sources = 
 
-noinst_PROGRAMS = $(TEST_PROGS) test-interactive test-interactive-multi
+check_PROGRAMS = $(TEST_PROGS) test-interactive test-interactive-multi
 
 test_interactive_SOURCES	= test-interactive.c $(helper_sources)
 test_interactive_LDADD		= $(gtk_hotkey_libs)

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWSAaPjsAA8nfgCAQVeP////v
//C////wYAdO+NAAAVBAKAUBCSUFN6hPRtTQKeEZNMFNPAmpibUemp6aR6QNMcZMmhiMTRgEYCYQ
BgJpo0yNAMcZMmhiMTRgEYCYQBgJpo0yNAMNQ1QGmgD9SGjQADQAAaAAABxkyaGIxNGARgJhAGAm
mjTI0AwVSJoEZABNACaNGmp6jFNPE01A8mRkaak8FMVHyj32iXygPTaczN6qyTiG1KZfCgKQGSiG
OWlqIIBvCSpFlIfBAfpQBIMkEFGoOgVh2KCxmROVQkbVvKXLEkpSlHz7HKj5vpYuasZVb0lUqKJv
boyMrwPzKnrRR/5utzPnrf2+M0pJxOl31rPkkUwSdWHzStsLpHtY7dk8PU6Xmci13U55Su10nVSr
qIpTKVyGJCMymGHsZxmu+xegr6AQZRZBUBXCsgghDoMVGmEcuR/DiR25xxrSe3jvRj9YUlDr9Ajx
bOB+z186/2RDDdlmnOePk07I2MSOKjzwjbT2Ld6BlU3qZEoWosd6Mq20vygyW9qTxoy1sSdPsPCp
i/Fbc0Gv9WhFEojT2O5FUe9tho24JlUIZzN4Jwhfbg5dxhkYJxkJtqjUmrymWEURU32Cp2bWirMM
FciuuzW4lGc17F0nEtTBa0SV0vLmhpFWBmvYJ+eqEZZDar2rm7roZsF0HpqbWycFMxajOzLqNVia
iUrE6mn7DIm4GR5oRRmtjNCJH8s6oamTZgi1SmE7YR5TejLVg27aI3NutrpOVhdEW5YRctX6aZzN
mNUIqYZ8/20VNLcnXCMyktO5GlzKV6BhfWaVF9v8tWmxRWk3sUodHnbrVc42tX21yisyj3eoxX3y
SJ7krHI1zk7Bxa10Iz4zRot1sRgq4KmplLB6LVr0jjzLYRlYRDKxgKrjFc5FViizWoo1tDI0tJP9
bFFTMryN1PJX2quVY8hwTRkS0tUItF+C1YyZHLceFrkhGUvcbB3r1rrxVyWF1xIkKuMwOKCz0mh6
jnxlMj3JQ8bOzrjoRwVTiV5rklKFznbjc9CpcmqktSVKR4UiIYQ+Lf+D2+/J/HlfT2PeeDh8nc7s
qlgZmEyN4fejq5T4TlIFXhnjHntRqf2qOGJpB8K9+JqfV/XS13vPzO7c2GtH0Vo/pOrvU6H99W19
8PD4XuU8Wnr6AoU7PkzyTB31INJGa/h+9RzhSC2MdjGHqY0XKkYm8ykKV5Y7ePA8KHqFYv4C53Zy
tmn58BMBaVwyZZxjlB16gWw78RrxFepf23uNUpjacGYg03307sOYmE8gQdcJj3RxMzEvE3DuYRvs
w74/NCbRGRkJo3DtXcrBOyelfWRbN5YCJBsRhGDx4CuIthdElOMKkEoMZuMkLnp6HmiI5edUyu50
P2sc3vWW7ORJ+k/dP4GAvnGyU+xeqVJ4sr3z58zwRJNPnRNTm3tWe/JU6bIexuRW0o7bGrhObzmF
NcUZoVKCZwqeJnXyiExR6NbO79i40ydZmq/13jzXPR4SmjolKWD748yVDBUxp0tXI2cK+lwfg7Yf
wVuKTvPhpVwZsn/XdxN215MSbk0nBt8IhfejWrZxHIiZGTbsOP2pREPg5UPFDKWIkk+ORBMZ1Gtz
+Hq3NaSXBOO1401XW6b+dKMvUaj6KkeZ3uRLh0zGom3uUTIRnaZC/6SNFiY/MSHjj/Evxcaik8hY
R0vh8e8rdXqnX9fXfiiXdNvyRLF9GVD70uOUxdsOSG7pIsgc+uDPi+XxdaixgQx8EGZ+TBjFjZ4y
ri4R6zgnd2Q3wRvRuhSOEHVD2KOEIzCkO2GkfrhSIeok/A93Hjn6DtbOIq93PuWo90KQjddDB6Vs
LYip1NB5bJR0sq1UVwicOE5qOLPOIVREPxmNkMyi2CijgqvfzJkRXfMrSRESkR8k3LJH5dyOzARu
3dcRa+UkHpdgiuUkLpwdYicR1vXRGGFI/enI44fOzdHQ6HOLTvsQxvyOHIZRSWxW2xS9NGvAXTGw
3KAjQbF1PMo4SeYw3taSKl4jJbbOCKojZTYhRRIlJFtcKkbjGulR45ynUqJLdXdTQdSLTj/tpjN5
SdCUVVRPx6lZsay1ZU4kdaRFbMLlmoi+DGE06XNv352o3POtJcnFY/2WkjOqbWxC4lLDEqP0hHEz
VpXCP8YQTRl451REjJIb3JqfvWjrGtFr4/mjydR5jmvRm0L4943ufUURNBvR4eLhbvhHkaRkd1+X
GyvAXDOriqbm6B8IRXFVdbSGrahU+hc6pCce4u5IpwoSBANHx2A=
